src/main/java/backend/resource/MultiModel.java

```diff
@@ -49,6 +49,12 @@ private synchronized MultiModel add(Model model) {
         return this;
     }
 
+    public synchronized MultiModel removeRepoModelById(String RepoId) {
+        Optional<Model> repoModelToBeDeleted = getModelById(RepoId);
+        this.models.remove(repoModelToBeDeleted.get().getRepoId());
```

Comment:
Yeah, it's resolved. I used the `Model` instead of the repoId. Okay, will do a check.